tests/update/updates_packagekit_kde.pm

```diff
@@ -86,6 +86,8 @@ sub run {
         }
         # Close tray updater
         send_key("alt-f4");
+        # Mouse click to close in case "alt-f4" doesn't work sometimes
+        click_lastmatch if check_screen("updates_packagekit_kde-up-to-date", 5);
```

Comment:
This is racy and will trigger even if alt-f4 works. You'd need some `check_screen("tray-closed")` needle or similar.